src/app/pages/color_schemes/mod.rs

```diff
@@ -409,7 +437,10 @@ impl ColorSchemes {
 #[derive(Deserialize, Serialize, Debug, Clone, PartialEq)]
 pub struct ColorScheme {
     pub name: String,
-    pub theme: ThemeBuilder,
+    pub theme_builder: ThemeBuilder,
+    // xxx: should we not serialize theme ?
+    // building it is costly, but this struct my change over time
```

Comment:
If we construct it each time, we would just have to avoid blocking the main thread if the delay is significant.

Reply:
I think I understood that backwards, let's serialize it if we can avoid the resize issue, we would just have to handle errors in deserialization when the struct changes and default the missing fields.